### ISSUE TITLE
Planck like simulations

### ISSUE THREAD

**Issue by thibautlouis** [simonsobs/map_based_simulations#11]
For the LAT likelihood, we would like to test co-addition with Planck data. So we would need Planck-like simulations. Maybe we should start by generating foregrounds simulations corresponding to Planck frequency. Having Planck noise simulations would also be useful.

**Comment by zonca** [comment]
ok, so nside 4096, delta bandpasses at Planck frequencies, gaussian beams, both LFI and HFI.
All foregrounds components currently available:

* dust, synchrotron, free-free, ame with spatially fixed spectral index
* CIB, ksz, tsz
* CMB 

Is this ok?
This will go after the time domain simulations, so after the end of June

**Comment by msyriac** [comment]
It should also be straightforward to take the existing SONoiseSimulator class in https://github.com/simonsobs/mapsims/blob/master/mapsims/noise.py and extend it to take as input (a) white noise levels for Planck and (b) the hitmaps for each array. We need a volunteer for this.

**Comment by zonca** [comment]
yes @msyriac create another class similar to `SONoiseSimulator` for Planck

**Comment by zonca** [comment]
signal maps with Dirac delta bandpasses available, see https://github.com/simonsobs/map_based_simulations/issues/24
